torchbiggraph/config.py

```diff
@@ -8,14 +8,14 @@
 
 import argparse
 import importlib
+from itertools import chain
```

Comment:
I understand what you were trying to do here, but the rule (followed by Black and isort) is to put all `import foo` statements before all `from foo import bar` statements, and only sort within each block.